src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml

```diff
@@ -120,7 +128,7 @@
                 </DataGridTextColumn>
                 <!-- Package Version -->
                 <DataGridTextColumn 
-                    Header="Version" 
+                    Header="{x:Static w:Resources.VersionHeaderText}" 
```

Comment:
thanks

Reply:
Thanks, this should fix the localization bug